app/src/main/res/values/strings.xml

```diff
@@ -72,7 +72,7 @@
     <string name="main_not_numeric">%1$s is not a number.</string>
     <string name="main_invalide_trainerlvl">Invalid Trainer Level!</string>
     <string name="notification_title">GoIV Running - Level %1$s</string>
-    <string name="notification_text">Tap to open</string>
+    <string name="notification_open_app">Open app</string>
```

Comment:
Minor comment: why do we need to change this string (hence retranslate it) if the meaning is the same? Is it a matter of space?